### ISSUE TITLE
V7 Client Bug - context active_model and active_ids

### ISSUE THREAD

**Issue by richard-willowit** [odoo/odoo#347]
When chaining to a view via actions, the client retains a context that is in line with the starting model, thus presenting a context inconsistent with true modular coding.

For example, choosing a customer, then "More / Invoices" and then choosing any object based action (for example Send by Email, or Register Payment) the context passed by the client has active_id and active_model for res.partner not account.invoice.  (The id passed to the action is correct, but there are occasions when the active_model is needed...)

**Comment by Yenthe666** [comment]
Dear @richard-willowit,

Thank you for reporting this issue!
At this point Odoo 6.0, 6.1 and 7.0 are no longer supported for bugsfixes anymore, which means we will no longer handle these bugs.
If you still face this issue on Odoo 8, 9 or 10 we'd like to ask you to create a new issue on how to reproduce this.
Our apologies.

Regards,
Yenthe

**Comment by richard-willowit** [comment]
@Yenthe666 - You may note this was reported over 2 years ago, and was not dealt with then.  I am not relying or waiting on a fix here anymore, but it is a pity that it was not addressed when it was an issue in the "current" version.

**Comment by Yenthe666** [comment]
Hi @richard-willowit,

I agree, it is a pity that there hasn't ever been dealt with this issue. The problem back then (and it still is) is that all Odoo issues that are monitored or are fixed are done by a few people that try to do it side by side with their job.
I for example try to keep issues in control since a while but I don't get paid for it and it is out of goodwill, so there is only so much I can do.
Nevertheless, I'm sorry it didn't get fixed back then. If you ever create a new issue please ping me and I'll make sure it will get adressed.

Regards,
Yenthe
